apps/lend/src/components/DetailsUser/components/DetailsUserSupply.tsx

```diff
@@ -50,7 +50,7 @@ const DetailsUserSupply = (pageProps: PageContentProps) => {
       ) : foundVaultShares ? (
         <ContentWrapper paddingTop>
           <Wrapper>
-            <CellUserMain {...pageProps} type="supply" />
+            <CellUserMain {...pageProps} market={market!} type="supply" />
```

Comment:
🙈

Reply:
so the pageProps are defined globally, but in this case we know the market is set, so it's safe to use `!`
however, it would be best to refactor and remove market from pageProps